packages/app/src/App.tsx

```diff
@@ -69,6 +69,9 @@ const app = createApp({
     bind(catalogPlugin.externalRoutes, {
       createComponent: scaffolderPlugin.routes.root,
     });
+    bind(explorePlugin.externalRoutes, {
+      catalogEntity: catalogPlugin.routes.catalogEntity,
```

Comment:
Part of the trickiness that we wanted to solve with binding parameterized external route refs is to add the possibility to map parameters to the target route in case there's not an exact match in the parameter name. Would either want to implement it at the same time, or ensure that we're able to do it.

Thinking something like

```ts
bind(explorePlugin.externalRoutes, {
  catalogEntity: {
    target: catalogPlugin.routes.catalogEntity,
    mapParams:  ({otherName, ...rest}) => ({name: otherName, ...rest}),
  },
})
```

Reply:
Make sense, I will have a look when updating this.